medusa/search/manual.py

```diff
@@ -265,7 +273,43 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
         # give the CPU a break and some time to start the queue
         time.sleep(cpu_presets[app.CPU_PRESET])
     else:
-        provider_results['found_items'] = sql_total
+        cached_results = [dict(row) for row in sql_total]
+        for i in cached_results:
+            i['quality_name'] = Quality.splitQuality(i['quality'])
+            i['time'] = datetime.fromtimestamp(i["time"]).strftime(app.DATE_PRESET + " " + app.TIME_PRESET)
```

Comment:
This for loop has a mix of double quotes and single quotes. It's just cosmetic, but still, it's a quick fix.